NewChallenge2.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>New Challenge 2</title>
+</head>
+
+<body>
+    <h1>New Challenge 2</h1>
+    <div>
+        <h2>What chore needs to be done</h2>
+        <input class="chore" type="text" placeholder="chore" value="">
+        <button class="choreBtn">submit</button>
+    </div>
+    <div>
+        <h2>Enter name</h2>
+        <input class="name" type="text" placeholder="name" value="">
+        <button class="nameBtn">submit</button>
```

Comment:
You've got inconsistent naming in this file for your classes. Rather stick to hyphens in HTML files.

Rule:
CamelCase for Javascript
Hyphens for HTML